src/Root.h

```diff
@@ -208,6 +210,7 @@ class cRoot
 
 	cCraftingRecipes * m_CraftingRecipes;
 	cFurnaceRecipe *   m_FurnaceRecipe;
+	cBrewingRecipes *  m_BrewingRecipes;
```

Comment:
Okay this result in changing around 10 lines and still not works. Let it for a own pull request...